locale_test.go

```diff
@@ -75,6 +75,7 @@ func TestDetect(t *testing.T) {
 		expectError error
 	}{
 		{"normal", []string{"en-US"}, nil, language.AmericanEnglish, nil},
+		{"invalid", []string{"ac"}, nil, language.Und, nil},
```

Comment:
Added a testcase that would return an error if doing `language.Parse` instead of `language.Make`, to make it define the current behavior